support-frontend/assets/pages/paper-subscription-landing/components/paperPrices.tsx

```diff
@@ -40,7 +40,30 @@ const getPriceCopyString = (
 	return <>per month{productCopy}</>;
 };
 
+const autumnPromoCodes = [
+	'AUTUMNEVERYDAY10HD',
+	'AUTUMNSAT10',
+	'AUTUMNSAT10HD',
+	'AUTUMNSIXDAY10',
+	'AUTUMNSIXDAY10HD',
+	'AUTUMNSUN10',
+	'AUTUMNSUN10HD',
+	'AUTUMNWEEKEND10',
+	'AUTUMNWEEKEND10HD',
+	'AUTUMN14',
+	'FESTIVAL20',
+	'AUTUMN20',
+	'AUTUMN30',
+];
+
 const getOfferText = (price: ProductPrice) => {
```

Comment:
is this called for each price card? I'm trying to understand why there are multiple `promotions`

Reply:
They needed multiple promotions (one for each package plus some with higher discounts). So I'm checking whether any of the promos applied to the package is from this campaign.